content/technical-guide/third-party-libraries/cawemo/cawemo-backend.md

```diff
@@ -73,19 +73,21 @@ This section covers third-party libraries used by the Cawemo backend. All of the
 - [javax.persistence:javax.persistence-api@2.2](https://github.com/javaee/jpa-spec) ([Eclipse Public License v1.0](http://www.eclipse.org/legal/epl-v10.html))
 - [javax.xml.bind:jaxb-api@2.3.1](https://github.com/javaee/jaxb-spec/jaxb-api) ([CDDL 1.1](https://oss.oracle.com/licenses/CDDL+GPL-1.1))
 - [net.bytebuddy:byte-buddy@1.12.23](https://bytebuddy.net/byte-buddy) ([Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt))
+- [net.java.dev.jna:jna@5.13.0](https://github.com/java-native-access/jna) ([LGPL-2.1-or-later](https://www.gnu.org/licenses/old-licenses/lgpl-2.1))
 - [org.apache.commons:commons-lang3@3.12.0](https://commons.apache.org/proper/commons-lang/) ([Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt))
 - [org.apache.httpcomponents:httpclient@4.5.14](http://hc.apache.org/httpcomponents-client-ga) ([Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt))
 - [org.apache.httpcomponents:httpcore@4.4.16](http://hc.apache.org/httpcomponents-core-ga) ([Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt))
 - [org.apache.logging.log4j:log4j-api@2.17.2](https://logging.apache.org/log4j/2.x/log4j-api/) ([Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt))
 - [org.apache.logging.log4j:log4j-to-slf4j@2.17.2](https://logging.apache.org/log4j/2.x/log4j-to-slf4j/) ([Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt))
 - [org.apache.tomcat.embed:tomcat-embed-el@9.0.83](https://tomcat.apache.org/) ([Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt))
 - [org.apiguardian:apiguardian-api@1.1.2](https://github.com/apiguardian-team/apiguardian) ([The Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt))
+- [org.aspectj:aspectjrt@1.9.7](https://www.eclipse.org/aspectj/) ([Eclipse Public License - v 2.0](https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt))
```

Comment:
Hm, `jna` and `aspectjrt` should not show up in the list actually. The former is a test-only dependency and the latter is not used at all.

Reply:
Should I do something before running the make target?
The file generated by `make generate-restapi-enterprise-dependency-license` includes these dependencies. I just checked again.

Reply:
That's odd, because for me it doesn't include them 🤔.

Do you have a "Reload Maven project" option or so in VS Code?
Can you also check which Maven version is used (by running `mvn -v`)?

Reply:
Ok, it's the same for me 🤷🏻‍♂️. The only difference is the macOS version and architecture (but that shouldn't matter).

Reply:
I tried deleting the contents of the `.m2/repository` folder and installing dependencies again. Still getting these dependencies in the output of the make target.

I installed the maven extension for VSC but it does not have any option to "Reload Maven Project". There was an option to "Show Effective POM" (do not exactly know what it is) and this `EffectivePOM.xml` includes the `aspectjrt` dependency.

However, if I run `mvn dependency:tree`, I do not see `aspectjrt`.

I don't think this is worth more of my time now. Is it possible for you to make a commit with the correct dependencies for `restapi` so that we can unblock this PR for now and get the release out the door?

Reply:
> I don't think this is worth more of my time now. Is it possible for you to make a commit with the correct dependencies for restapi so that we can unblock this PR for now and get the release out the door?

Sure, I can add a commit.

Reply:
It were actually just these two dependencies that had to be removed.